src/api/resources/sync/views.py

```diff
@@ -112,6 +113,24 @@ def _get_source(source_type):
     return project_sources.get(source_type) or non_project_sources.get(source_type)
 
 
+def _get_project(data, source_type):
+    projid = None
+    projname = ""
```

Comment:
If this is getting display at some point, should there be a placeholder name, in the case the project doesn't exist?

Reply:
good point